### DISCUSSION TITLE
Kibana and some other Processes are down

### DISCUSSION THREAD

**Discussion by 4TLA** [telekom-security/tpotce#1114] [General]
I just installed Tpot on a google cloud VM for the 5th time in a row. Every time when i install it the same processes are down when i view /opt/tpot/bin/dpsh.sh 

sudo: unable to resolve host prettyadvertising: Name or service not known
[ ========| System |======== ]
     DATE:  Sun Jun 12 16:31:34 UTC 2022
   UPTIME:   16:31:34 up 1 min,  1 user,  load average: 15.05, 4.29, 1.47
    T-POT:  ACTIVE       
BLACKHOLE:  DISABLED
NAME                  STATUS                       PORTS
adbhoney              Up 14 seconds (health: starting)   0.0.0.0:5555->5555/tcp
ciscoasa              Up 16 seconds                      0.0.0.0:5000->5000/udp, 0.0.0.0:8443->8443/tcp
citrixhoneypot        Up 17 seconds                      0.0.0.0:443->443/tcp
conpot_guardian_ast   Up 17 seconds (health: starting)   0.0.0.0:10001->10001/tcp
conpot_iec104         Up 41 seconds (healthy)            0.0.0.0:161->161/udp, 0.0.0.0:2404->2404/tcp
conpot_ipmi           Up 15 seconds (health: starting)   0.0.0.0:623->623/udp
conpot_kamstrup_382   Up 24 seconds (health: starting)   0.0.0.0:1025->1025/tcp, 0.0.0.0:50100->50100/tcp
cowrie                Up 40 seconds                      0.0.0.0:22-23->22-23/tcp
ddospot               Up 15 seconds                      0.0.0.0:19->19/udp, 0.0.0.0:53->53/udp, 0.0.0.0:123->123/u
dp, 0.0.0.0:1900->1900/udp
dicompot              Up 17 seconds                      0.0.0.0:11112->11112/tcp
dionaea               Up 16 seconds (health: starting)   0.0.0.0:20-21->20-21/tcp, 0.0.0.0:42->42/tcp, 0.0.0.0:81->
81/tcp, 0.0.0.0:135->135/tcp, 0.0.0.0:445->445/tcp, 0.0.0.0:1433->1433/tcp, 0.0.0.0:1723->1723/tcp, 0.0.0.0:1883->1
883/tcp, 0.0.0.0:3306->3306/tcp, 0.0.0.0:27017->27017/tcp, 0.0.0.0:69->69/udp
elasticpot            Up 17 seconds                      0.0.0.0:9200->9200/tcp
elasticsearch         Up 20 seconds (health: starting)   127.0.0.1:64298->9200/tcp
ewsposter             Up 25 seconds                      
fatt                  Up 43 seconds                      
heralding             Up 21 seconds                      0.0.0.0:110->110/tcp, 0.0.0.0:143->143/tcp, 0.0.0.0:465->4
65/tcp, 0.0.0.0:993->993/tcp, 0.0.0.0:995->995/tcp, 0.0.0.0:1080->1080/tcp, 0.0.0.0:5432->5432/tcp, 0.0.0.0:5900->5
900/tcp
honeytrap             Up 44 seconds                      
ipphoney              Up 34 seconds                      0.0.0.0:631->631/tcp
mailoney              Up 24 seconds                      0.0.0.0:25->25/tcp
map_redis             Up 29 seconds                      
map_web               Up 42 seconds                      127.0.0.1:64299->64299/tcp
medpot                Up 30 seconds                      0.0.0.0:2575->2575/tcp
nginx                 Up 44 seconds                      
p0f                   Up 43 seconds                      
redishoneypot         Up 15 seconds                      0.0.0.0:6379->6379/tcp
sentrypeer            Up 18 seconds                      0.0.0.0:5060->5060/udp
snare                 Up 1 second                        0.0.0.0:80->80/tcp
spiderfoot            Up 28 seconds (health: starting)   127.0.0.1:64303->8080/tcp
suricata              Up 44 seconds                      
tanner                Up 5 seconds                       
tanner_api            Up 10 seconds                      
tanner_phpox          Up 18 seconds                      
tanner_redis          Up 17 seconds                      
kibana                DOWN                  
logstash              DOWN
map_data              DOWN

I installed tpot with the command `./install.sh --type=user` as root. Also i updated and upgraded apt.

**Comment by t3chn0m4g3** [comment]
[Duplicate](https://github.com/telekom-security/tpotce/discussions/1089).
